src/python/WMComponent/AgentStatusWatcher/AgentStatusPoller.py

```diff
@@ -550,8 +550,8 @@ def uploadToAMQ(self, docs, producer, agentUrl, timeS):
                                 host_and_ports=self.hostPortAMQ,
                                 logger=logging)
 
-            notifications = stompSvc.make_notification(payload=docs, docType=docType,
-                                                       docId=producer, ts=timeS)
+            notifications = stompSvc.make_notification_list(payload=docs, docType=docType,
```

Comment:
Benjamin, can you also update these 2 calls such that they reflect your changes to StompAMQ module?
Something like
```
notifications = []
for doc in docs:
    notifications.append(make_notification(payload=doc, docType=docType, ts=timeS, dataSubfield="payload"))
```

Same for the module below. Thanks

Reply:
Sigh.. you missed the "dataSubfield" bit and I didn't catch it on my review either. Fixing it in a new PR